addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -184,6 +189,22 @@ class Sanitize {
                 node.textContent = '\u200B';
             }
 
+            // Ensure the editor tabs allign of the 40px grid
+            if (isEditorTab(node)) {
+                let tabPreviousSibling = node.previousSibling;
+                while (isZWS(tabPreviousSibling)) {
+                    tabPreviousSibling = tabPreviousSibling.previousSibling;
+                }
+                if (isEditorTab(tabPreviousSibling)) {
+                    node.style.width = '40px';
+                } else {
+                    const editable = closestElement(node,'.odoo-editor-editable', true);
+                    if (editable && editable.firstElementChild) {
+                        alignNodeOnGrid(node, editable.firstElementChild);
```

Comment:
This is only used once and is ultra specific to the particular behavior at hand here. I would inline the function here and precede it with a comment that says "// Align node on grid.". If we ever do need to align other nodes on the grid, we can make an util function then.

Reply:
:nby: :grimacing: